katalog/nginx/MAINTENANCE.md

```diff
@@ -1,156 +1,93 @@
+1. Go to the [NGINX ingress controller repository](https://github.com/kubernetes/ingress-nginx/) and check the latest release.
 
-Search for the version of the chart that installs your desired version of the controller (for example, chart v4.11.2 installs the controller v1.11.2) and run the following command:
+> Controller releases are published as `controller-vX.Y.Z` tags.
 
-> ❗️ change the `--version` flag with the right verison.
->
-> You can check the version of the controller in Ingress NGINX Controller's [Supported Versions Table](https://github.com/ralgozino/ingress-nginx?tab=readme-ov-file#supported-versions-table).
+2. Helm template the content using the following command:
 
 ```bash
 helm template ingress-nginx ingress-nginx \
   --repo https://kubernetes.github.io/ingress-nginx \
-  --version 4.11.2 \
-  --namespace ingress-nginx \
-  --set fullNameOverride=ingress-nginx \
-  --set controller.kind=DaemonSet \
-  --set controller.image.registry=registry.sighup.io \
-  --set controller.image.image=fury/ingress-nginx/controller \
-  --set controller.image.pullPolicy=Always \
-  --set controller.image.allowPrivilegeEscalation=true \
-  --set controller.containerPort.http=8080 \
-  --set controller.containerPort.https=8443 \
-  --set controller.allowSnippetAnnotations=true \
-  --set controller.service.type=NodePort \
-  --set controller.service.nodePorts.http=31080 \
-  --set controller.service.nodePorts.https=31443 \
-  --set controller.service.externalTrafficPolicy=Local \
-  --set controller.admissionWebhooks.port=9443 \
-  --set controller.admissionWebhooks.service.servicePort=9443 \
-  --set controller.admissionWebhooks.patch.enabled=false \
-  --set controller.admissionWebhooks.certManager.enabled=true \
-  --set controller.admissionWebhooks.certManager.enabled=true \
-  --set controller.metrics.enabled=true \
-  --set controller.metrics.serviceMonitor.enabled=true \
-  --set controller.metrics.serviceMonitor.additionalLabels.k8s-app=ingress-nginx \
-  --set controller.metrics.serviceMonitor.scrapeInterval=10s \
-  --set controller.metrics.serviceMonitor.jobLabel=k8s-app \
-  --set controller.metrics.prometheusRule.enabled=true \
-  --set controller.metrics.prometheusRule.additionalLabels.prometheus=k8s \
-  --set controller.customTemplate.configMapName=nginx-configuration \
-  --set controller.config.enable-access-log-for-default-backend=true \
-  --set controller.config.http-redirect-code=301 \
-  --set controller.config.proxy-body-size=10m \
-  --set controller.updateStrategy.type=RollingUpdate \
-  --set controller.updateStrategy.rollingUpdate.maxSurge=1 \
-  --set controller.updateStrategy.rollingUpdate.maxUnavailable=0 \
-  --set controller.podSecurityContext.fsGroup=101 \
-  --set controller.ingressClassResource.default=true \
-  > templated-from-helm.yaml
+  --namespace ingress-nginx --create-namespace --version $VERSION --values MAINTENANCE.values.yml | yq -s '.kind + "-" + .metadata.name'
```

Comment:
TIL `yq -s`, nice!

It is a good practice to use the long-form of flags in scripts and documentation, for better readability:

```suggestion
  --namespace ingress-nginx --create-namespace --version $VERSION --values MAINTENANCE.values.yml | yq --split-exp '.kind + "-" + .metadata.name'
```